_data/projects/InTalks.yml

```diff
@@ -0,0 +1,12 @@
+name: InTalks
+desc: InTalks is an android application used to fetch the latest news about a topic from various online news platforms.
+site: https://www.github.com/josharsh/Intalks
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Java
+- Android
+- API
+
+upforgrabs:
+  name: good first issue
+  link: https://github.com/josharsh/InTalks/blob/master/README.md
```

Comment:
Can this link point to a list of curated issues? I don't see any created in the repository currently...